tiatoolbox/utils/misc.py

```diff
@@ -154,18 +153,18 @@ def imwrite(image_path: os | PathLike, img: np.ndarray) -> None:
 
     """
     if isinstance(image_path, Path):
-        image_path = str(image_path)
+        image_path_str = str(image_path)
```

Comment:
what if image_path is not a Path?

I think you can remove the if statement here and just run str without check.